foreman_ansible_inventory.py

```diff
@@ -107,6 +110,15 @@ def read_settings(self):
         self.foreman_user = config.get('foreman', 'user')
         self.foreman_pw = config.get('foreman', 'password')
         self.foreman_ssl_verify = config.getboolean('foreman', 'ssl_verify')
+        try:
+            self.foreman_prefix_json = config.get('foreman', 'prefix_json')
```

Comment:
Sorry, I don't follow why you need to provide a prefix. Why not just try to parse the parameters as JSON or YAML  - and if that doesn't work, default to string?